test/libsolidity/syntaxTests/types/encoding_packed_fractional.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+   function f1() public pure returns (bytes memory) {
+       return abi.encodePacked(0.1, 1);
+   }
+}
+// ----
+// TypeError: (98-101): Fractional numbers cannot yet be encoded.
+// TypeError: (103-104): Cannot perform packed encoding for a literal. Please convert it to an explicit type first.
```

Comment:
Hm, this kind of worries me that we have such different codepaths. Also this message is a bit confusing for the above case.

Reply:
Not sure what you mean. This message is about the `1`, not the `0.1`.

Reply:
Yep it's about the `1`.

Reply:
Ah ok, it is not an issue if the source location is nicely formatted. Just in the tests :)